### DISCUSSION TITLE
Problem using QSkinny in qml

### DISCUSSION THREAD

**Discussion by hestrro** [uwerat/qskinny#497] [Q&A]
![image](https://github.com/user-attachments/assets/fb3dbc5e-9aa2-44d8-9ef5-ea74de37f2da)
set(CMAKE_PREFIX_PATH "D:/qsk1206/lib/cmake")
find_package(Qt6 REQUIRED COMPONENTS Core Quick)
find_package(QSkinny REQUIRED)
target_link_libraries(TreeView PRIVATE
    Qt6::Core
    Qt6::Quick
    Qsk::QSkinny
     Qsk::QmlExport
)

**Comment by hestrro** [comment]
![image](https://github.com/user-attachments/assets/9759073a-a36b-4cdc-9abf-dc54063bf78f)
This situation also occurs in QSkinny's example

**Comment by uwerat** [comment]
Shoud be a problem of your build/runtime environment - f.e the shapes module is from Qt ( not QSkinny )
